.github/workflows/build-release.yml

```diff
@@ -0,0 +1,68 @@
+name: Build Release
+
+on: [ push, pull_request ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+env:
+  CARGO_TERM_COLOR: always
+  PROTOC: protoc
+
+jobs:
+  build_release:
+    name: Build Release
+    runs-on: ${{ matrix.config.os }}
+    strategy:
+      matrix:
+        config:
+          - { os: ubuntu-latest, target: x86_64-unknown-linux-gnu, cross: false }
+          - { os: ubuntu-latest, target: aarch64-unknown-linux-gnu, cross: true }
+          - { os: macos-11, target: x86_64-apple-darwin, cross: false }
+          - { os: macos-11, target: aarch64-apple-darwin, cross: false }
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - if: ${{ matrix.config.os == 'ubuntu-latest' }}
+        run: sudo apt install zlib1g-dev
+
+      - uses: arduino/setup-protoc@v1
+        with:
+          version: '3.x'
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          target: ${{ matrix.config.target }}
+          components: rustfmt, clippy
+
+      - uses: Swatinem/rust-cache@v1
+
+      - name: Build Databend
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          use-cross: ${{ matrix.config.cross }}
```

Comment:
cross in here would not work, we have investigated it before.
https://github.com/datafuselabs/databend/issues/1850